powerauth-webflow-authentication-login-sca/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/method/loginsca/controller/LoginScaController.java

```diff
@@ -234,6 +234,10 @@ protected AuthMethod getAuthMethodName() {
      */
     private void updateUsernameInHttpSession(String username) {
         synchronized (httpSession.getServletContext()) {
+            if (username == null || !username.matches("^[0-9a-zA-Z_]+$")) {
```

Comment:
Maybe username should also be able to contain `@`, `.`, `-`, `/`, `\` or any character that can be typed, to facilitate usernames represented via e-mail, "domain username" (such as `wultra:users/roman`), etc.